src/features/Header/Header.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './header.css';
+import Button from './Button';
+import Socials from './Socials';
+import selfImage from '../../assets/Image/selfImage.png';
+
+function Header() {
+  return (
+    <header id="header">
+      <div className="container header__container">
+        <h5>Hello I&apos;m</h5>
+        <h1>Krishna</h1>
+        <h5 className="text-light">Full Stack Developer</h5>
+        <Button />
+        <Socials />
+
+        <div className="myself">
+          <img src={selfImage} alt="Self" />
+        </div>
+        <a href="#contact" className="scroll__down">Scroll Down</a>
```

Comment:
![image](https://github.com/Krishnabot/Portfolio-React/assets/116451144/4f694710-7f9d-482f-b14f-d8f013821255)

- [ ] Wow, Awesome work here, but your image won't show up  here may be some responsive or some wrong z-indexing